bin/create-initial-roles-and-users.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+##This script will create the user roles.
+
+parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+cd "$parent_path"
+
+../bin/idam-create-caseworker.sh caseworker,caseworker-autotest1 auto.test.cnp@gmail.com password
```

Comment:
we don't need ../bin here. and, could we please split the role creations from user creations into 2 groups one after the other?

Reply:
Reorganized the file.